src/reactIntl.js

```diff
@@ -0,0 +1,10 @@
+export const intl = {
+	formatMessage: jest.fn(trn => trn.defaultMessage || 'formatted message'),
+	formatDate: jest.fn(date => date.toString() || 'formatted date'),
+	formatTime: jest.fn(time => time || 'formatted time'),
+	formatRelative: jest.fn(date => date.toString() || 'formatted relative time'),
+	formatNumber: jest.fn(num => num || 123),
+	formatPlural: jest.fn(str => str || 'formatted plural string'),
```

Comment:
this function takes a Number and returns a string https://github.com/yahoo/react-intl/wiki/API#formatplural